src/_scss/pages/search/searchPage.scss

```diff
@@ -15,6 +15,8 @@
     @import "resultsView/resultsView";
     @import "resultsView/dtuiOverrides";
 
+    @import "components/accordionCheckbox";
```

Comment:
is this import necessary?

Reply:
Yes, temp-page uses this file for its styling